pkg/gatewayserver/gatewayserver_util_test.go

```diff
@@ -97,6 +98,17 @@ func (is *mockIS) Get(ctx context.Context, req *ttnpb.GetGatewayRequest) (*ttnpb
 	return gtw, nil
 }
 
+func (is *mockIS) Update(ctx context.Context, req *ttnpb.UpdateGatewayRequest) (*ttnpb.Gateway, error) {
```

Comment:
Tests: change 4.

Maybe we should not ignore the FieldMasks and blindly update antennas.